app/views/users/migrate_token_confirmation.html.erb

```diff
@@ -1,4 +0,0 @@
-<%= form_tag migrate_token_path do %>
-  <%= hidden_field_tag "state", params[:state] %>
-  <%= submit_tag "Confirm that you've granted access", class: 'btn btn-primary' %>
-<% end %>
```

Comment:
This file is no longer being linked to, and I think all token migration is finished.